app/Models/Asset.php

```diff
@@ -106,7 +106,11 @@ public function declinedCheckout(User $declinedBy, $signature)
         'supplier_id'     => 'exists:suppliers,id|nullable',
         'asset_eol_date'  => 'date|nullable',
         'eol_explicit'    => 'boolean|nullable',
-        'byod'            => 'boolean',
+        'byod'            => 'nullable|boolean',
+        'order_number'    => 'string|max:191|nullable',
```

Comment:
Updating the `AssetFactory`'s definition to `'order_number' => (string) $this->faker->numberBetween(1000000, 50000000),` should fix the failing tests.

Reply:
Oh, awesome, thank you. The first one I'd looked at had something to do with `model_id` so I got too focused on that for a while, casting that to a string seems to have worked though. Thanks!